crates/matrix-sdk-indexeddb/src/crypto_store/mod.rs

```diff
@@ -138,6 +138,12 @@ impl From<indexed_db_futures::web_sys::DomException> for IndexeddbCryptoStoreErr
     }
 }
 
+impl From<serde_wasm_bindgen::Error> for IndexeddbCryptoStoreError {
+    fn from(e: serde_wasm_bindgen::Error) -> Self {
+        IndexeddbCryptoStoreError::Json(serde::de::Error::custom(e.to_string()))
```

Comment:
pre-existing: if that's not an API breaking change, could we rename the `Json` variant to `Serialization`?

Reply:
Sure! done in [a4c46b1](https://github.com/matrix-org/matrix-rust-sdk/pull/2884/commits/a4c46b14110b1df159551d82211476bc58ab1fc6)